Shared/css/style.css

```diff
@@ -1715,6 +1715,47 @@ input.hide-moments-button {
   float: left;
 }
 
+.filePopUp {
+  position: relative;
+}
+.fileUploadInfo {
+  display: none;
+  width: 100px;
+  position: absolute;
+  background: white;
+  padding: 10px 10px 0;
```

Comment:
It looks a good. I have some suggestion code for a little more centered look. The box wasnt centered when the text is english: 
![image](https://user-images.githubusercontent.com/38323901/57707559-20c57a80-7668-11e9-816f-6296001e29a5.png)


```suggestion
  padding: 10px 10px 0;
  border-radius: 5px;
  text-align: center;
  left: 5%;
```

Reply:
A quick fade in could be nice too, but not necessary